### DISCUSSION TITLE
IPVTL channels

### DISCUSSION THREAD

**Discussion by BellezaEmporium** [iptv-org/community#404] [Show and tell]
US IP (Bandwidth - poor):
http://190.122.96.187:8888/http/005 - NBC News / CBS New York
http://190.122.96.187:8888/http/002 - Azteca Honduras
http://190.122.96.187:8888/http/006 - Ve Plus Venezuela

BA IP (Bandwidth - average):
http://109.105.205.202:8888/http/001 - Prva Hercegovačka (Bosnia and Herzegovina)
http://109.105.205.202:8888/http/002 - Valentino Music HD
http://109.105.205.202:8888/http/003 - Valentino GTV
http://109.105.205.202:8888/http/004 - Valentino Etno

KX IP (Bandwidth - OK/Excellent): 
http://87.238.211.200:8888/http/001 - Turbo Channel Kosovo
http://87.238.211.200:8888/http/002 - MTV Kosova

DE IP (Bandwidth - OK/Excellent) : 
http://161.97.82.90:8888/http/001 - Antena 1 Romania
http://161.97.82.90:8888/http/003 - Pro TV Romania

AL IP (Bandwidth - OK/Excellent):
http://178.132.223.86:8888/http/001- Travelingo TV

IT IP (Bandwidth - Excellent):
http://89.42.134.226:8888/http/001 - Romance TV HD Germany

CN IP (Bandwidth - Average):
http://139.227.62.63:8888/http/001 - Phoenix Channel
http://139.227.62.63:8888/http/002 - CCTV 1
http://139.227.62.63:8888/http/003 - OTV Channel
http://139.227.62.63:8888/http/004 - Shanghai Media Dragon Channel
http://139.227.62.63:8888/http/005 - Hubei Radio & Television Station
http://139.227.62.63:8888/http/006 - Hunan TV
http://139.227.62.63:8888/http/007 - Zhejiang TV
http://139.227.62.63:8888/http/008 - Discovery Documentary China
http://139.227.62.63:8888/http/009 - SETV (東南衛視)
http://139.227.62.63:8888/http/010 - SiTV Laughter Theatre Channel 4K
http://139.227.62.63:8888/http/011 - SiTV City Channel
http://139.227.62.63:8888/http/012 - SiTV Animation Show Channel
http://139.227.62.63:8888/http/013 -SMG (Shanghai Media Group) Channel
http://139.227.62.63:8888/http/014 - ![image](https://user-images.githubusercontent.com/30985701/165118217-ed2f9b20-9067-4739-be8b-6d0b63c4270c.png)

**Comment by what-a** [comment]
http://190.122.96.187:8888/http/005
Looks like CBS New York (WCBS)
You're right about the bandwidth being low. It keeps cutting out for me. 
![image](https://user-images.githubusercontent.com/88639906/165198253-bba9fb75-1547-427b-97e3-e7791beb35cc.png)

**Reply by BellezaEmporium** [reply]
> http://190.122.96.187:8888/http/005 Looks like CBS New York (WCBS) You're right about the bandwidth being low. It keeps cutting out for me. ![image](https://user-images.githubusercontent.com/88639906/165198253-bba9fb75-1547-427b-97e3-e7791beb35cc.png)

Since it's people re-transmitting whatever they want using IPVTL, it might be possible that it has changed.

Yeah the bandwidth for this one is exquisitely... bad.

**Comment by KingMorisco** [comment]
> US IP (Bandwidth - poor): http://190.122.96.187:8888/http/005 - NBC News / CBS New York http://190.122.96.187:8888/http/002 - Azteca Honduras http://190.122.96.187:8888/http/006 - Ve Plus Venezuela
> 
> BA IP (Bandwidth - average): http://109.105.205.202:8888/http/001 - Prva Hercegovačka (Bosnia and Herzegovina) http://109.105.205.202:8888/http/002 - Valentino Music HD http://109.105.205.202:8888/http/003 - Valentino GTV http://109.105.205.202:8888/http/004 - Valentino Etno
> 
> KX IP (Bandwidth - OK/Excellent): http://87.238.211.200:8888/http/001 - Turbo Channel Kosovo http://87.238.211.200:8888/http/002 - MTV Kosova
> 
> DE IP (Bandwidth - OK/Excellent) : http://161.97.82.90:8888/http/001 - Antena 1 Romania http://161.97.82.90:8888/http/003 - Pro TV Romania
> 
> AL IP (Bandwidth - OK/Excellent): http://178.132.223.86:8888/http/001- Travelingo TV
> 
> IT IP (Bandwidth - Excellent): http://89.42.134.226:8888/http/001 - Romance TV HD Germany
> 
> ID IP (Bandwidth - Poor): http://87.238.211.200:8888/http/ ![image](https://user-images.githubusercontent.com/30985701/165107818-8bf21311-e10c-48cd-88c3-548b2cd92714.png)
> 
> CN IP (Bandwidth - Average): http://139.227.62.63:8888/http/001 - Phoenix Channel http://139.227.62.63:8888/http/002 - CCTV 1 http://139.227.62.63:8888/http/003 - OTV Channel http://139.227.62.63:8888/http/004 - Shanghai Media Dragon Channel http://139.227.62.63:8888/http/005 - Hubei Radio & Television Station http://139.227.62.63:8888/http/006 - Hunan TV http://139.227.62.63:8888/http/007 - Zhejiang TV http://139.227.62.63:8888/http/008 - Discovery Documentary China http://139.227.62.63:8888/http/009 - SETV (東南衛視) http://139.227.62.63:8888/http/010 - SiTV Laughter Theatre Channel 4K http://139.227.62.63:8888/http/011 - SiTV City Channel http://139.227.62.63:8888/http/012 - SiTV Animation Show Channel http://139.227.62.63:8888/http/013 -SMG (Shanghai Media Group) Channel http://139.227.62.63:8888/http/014 - ![image](https://user-images.githubusercontent.com/30985701/165118217-ed2f9b20-9067-4739-be8b-6d0b63c4270c.png)

how downolad that sir ?

**Reply by BellezaEmporium** [reply]
There's no way you can download those. It's .ts files, and so are only made for streaming.

**Reply by KingMorisco** [reply]
> There's no way you can download those. It's .ts files, and so are only made for streaming.

but how you found it this lis ?

**Reply by BellezaEmporium** [reply]
Censys, like the others.
